src/components/LoadingError/LoadingError.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+type Props = {
+  error: boolean;
+  setError: (parameter: boolean) => void,
+};
+
+export const LoadingError: React.FC<Props> = ({ error, setError }) => {
+  const hideError = () => {
+    setError(false);
+  };
+
+  if (error) {
+    setTimeout(() => {
+      setError(false);
+    }, 3000);
+  }
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className="notification is-danger is-light has-text-weight-normal"
```

Comment:
reread [task description](https://github.com/mate-academy/react_todo-app-loading-todos)
Be always attentive with tasks descriptions and checklists. There **no redundant** words. The same would be in test tasks. You need to read it attentively and write the code according to it. It is important.